src/Mod/PartDesign/Gui/Utils.cpp

```diff
@@ -90,9 +90,9 @@ bool setEdit(App::DocumentObject *obj, PartDesign::Body *body) {
     App::DocumentObject *parent = nullptr;
     std::string subname;
     auto activeBody = activeView->getActiveObject<PartDesign::Body*>(PDBODYKEY,&parent,&subname);
+    subname.clear();
     if (activeBody != body) {
```

Comment:
I'm also not understanding in which situation this statement is true. Anybody knows?

Reply:
I don't know too.